### PR TITLE
Snd 220

### PR DESCRIPTION
**Pull request by tkrsntr**
Payment Hub - CircleCI & optimisations (disabled Mojaloop)

### CODE REVIEW

**Review by VladislavEmilTodorov — APPROVED** [review]
OK, all needed steps that we did before manually are present.

**Comment by VladislavEmilTodorov** [comment]
Looks like that "ph-ee-operations-app" is still failing randomly. The "Fix Operation app DATABASE" step does not fix the racing condition problem.

**Comment by tkrsntr** [comment]
Added tryout function

**Review by jhyoty — APPROVED** [review]
one question, otherwise lgtm

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,26 +1,104 @@
...
-    # Add steps to the job
-    # See: https://circleci.com/docs/configuration-reference/#steps
+      - image: cimg/aws:2023.03
     steps:
       - checkout
+      - setup_remote_docker:
+          version: 20.10.14
+      - aws-cli/setup:
+          role-arn: 'arn:aws:iam::463471358064:role/CircleCIRole'
+          aws-region: AWS_DEFAULT_REGION
```
[the remark this answers is not in the document]
Comment (VladislavEmilTodorov) [reply .circleci/config.yml:18]: I think it is not. "aws-region" param has type "env_var_name".[ Setup params ](https://circleci.com/developer/orbs/orb/circleci/aws-cli#commands-setup)

**Pull request merged**